### ISSUE TITLE
Combining sensitivity maps

### ISSUE THREAD

**Issue by HartmutT** [jncc/bh3-automation#7]
This is partially verlaps #4.
The "Combining maps process" section on p. 3  of StepByStepMethod.docx states that "The area of output Species_sensitivity_Maximum was removed from area of output step 2 Species_sensitivity_Mode." A few lines below, it goes on to say that "Species_sensitivity_all_areas was erased from Habitat_sensitivity." The latter phrase clearly refers to the ArcGIS "Erase" tool. Does "removing the area" in the first sentence cited refer to the same tool? Without having seen the inputs to this steps, it would appear that an erase operation would wipe out all polygons unless the join to the species sample point only preserved polygons that have any species samples in them (inner join). 
Between the two steps cited above, "output step 1 Species_sensitivity_Maximum was merged with output step 2 " and at the end "The 3 maps were merged together." This question is the same as issue #4: is "merge" the equivalent of database UNION query, appending one record set to another without modifying the geomtries, or is some spatial operation to be performed on any overkapping geometries? This may be a non-issue if the previous procedure assures that there won't be any overlaps.

**Comment by LRPettit** [comment]
Yes, “removing the area” in the first sentence refers to the erase tool. The maps Species_sensitivity_Maximum and Species_sensitivity_Mode are the result of the process to join Species sample points with the habitat map. This process is done with the Spatial join tool and has to be repeated twice (once using setting merge rule for sensitivity to maximum, once setting merge rule to mode). Polygons with null values in the surface/subsurface species sensitivity columns (i.e. those polygons without species points) should be removed from these maps. Therefore the “combining maps process” uses a map with habitat sensitivities, and two maps with species sensitivities.

**Comment by LRPettit** [comment]
The merging process should not modify geometry and should not produce overlaps as the 3 sensitivity maps are supposed cover different areas. If overlapping polygons arise these should be removed as they might affect area calculation (to remove look at the “Removing Overlaps process” section in the StepbyStepMethod).

**Comment by LRPettit** [comment]
![image](https://user-images.githubusercontent.com/16681641/51172651-97fe2400-18ab-11e9-8f0d-07b926f4e095.png)
